apps/payments/next/app/layout.tsx

```diff
@@ -1,5 +1,5 @@
 import ExampleHeader from '../components/ExampleHeader';
-import './global.css';
+import 'libs/shared/styles/src/lib/tailwind.css';
```

Comment:
If you rebase to `main`, you should now be able to use package name imports, `@fxa/shared/styles` in this case.

That'd change the line to, `import '@fxa/shared/styles/lib/tailwind.css'` or something similar.